homeassistant/components/soma/__init__.py

```diff
@@ -72,6 +112,7 @@ def __init__(self, device, api):
         self.current_position = 50
         self.battery_state = 0
         self.is_available = True
+        self.api_is_available = True
```

Comment:
Why do we not set the entity as unavailable when the api is unavailable?

Reply:
It is something I have added with all the previous changes but you are totally right.. I reviewed it and it really is not useful to split in 2 "is_available" vars.
Since this PR is now merged, I guess I should submit a new one to clean this up?

Reply:
Yes. Please adjust in a new PR 👍